src/storage/storage.cc

```diff
@@ -245,8 +249,7 @@ Status Storage::Open(bool read_only) {
 
   std::shared_ptr<rocksdb::Cache> shared_block_cache;
   if (config_->rocks_db.share_metadata_and_subkey_block_cache) {
```

Comment:
@Yangsx-1 We can remove the condition check here since it makes sense to separate the cache by the column family.

Reply:
I think it's necessary. If the user manually set `rocks_db.share_metadata_and_subkey_block_cache` to `no`, the metatable cache size should be `metadata_block_cache_size`, which is assigned later. The same as subkey cache.

Reply:
Yes, but it makes no sense for users to separate the column family block cache, so it's good to ignore the share block cache condition.

Reply:
OK.  :D